repairnator/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/pipeline/Launcher.java

```diff
@@ -243,6 +250,24 @@ private void initNotifiers() {
         this.notifiers.add(new FixerBuildNotifier(notifierEngines));
     }
 
+    private List<String> getListOfProjectsToIgnore() {
+        List<String> result = new ArrayList<>();
+        if (this.config.getProjectsToIgnoreFilePath() != null) {
+            File file = new File(this.config.getProjectsToIgnoreFilePath());
+            try {
+                BufferedReader reader = new BufferedReader(new FileReader(file));
```

Comment:
Actually this code can really be simplified by using `Files#readAllLines`: https://docs.oracle.com/javase/8/docs/api/java/nio/file/Files.html#readAllLines-java.nio.file.Path-

See an example of this usage here: https://github.com/Spirals-Team/repairnator/blob/4566d6dc73f96acc0de686acccdaf9e8b4c61c44/repairnator/repairnator-realtime/src/main/java/fr/inria/spirals/repairnator/realtime/RTScanner.java#L89